src/main/java/gregtech/client/model/IModelSupplier.java

```diff
@@ -1,10 +1,14 @@
 package gregtech.client.model;
 
 import net.minecraftforge.client.event.TextureStitchEvent;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
 
 public interface IModelSupplier {
```

Comment:
`SideOnly(Side.CLIENT)` on the class, then there's no need to populate it for all methods.

Reply:
when a class implement a SideOnly interface and the class itself is used on the server side...... crash.

Reply:
Forge is not smart enough to strip them out? That's sad.